app/models/claim.rb

```diff
@@ -88,7 +88,7 @@ class Claim < ApplicationRecord
   validate :bank_account_number_must_be_eight_digits
   validate :bank_sort_code_must_be_six_digits
 
-  validate :claim_must_not_be_ineligible, on: :submit
+  validate :claim_must_be_eligible, on: :submit
```

Comment:
So technically "eligible" isn't the opposite of "ineligible" – a claim without all the answers to the eligibility questions won't be "ineligible", but that doesn't make it "eligible". The double negative is deliberate so as not to report a incomplete claim as "eligible".

Reply:
Ah ok. Makes sense. I'll remove it.